hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -614,6 +615,57 @@ def hpat_pandas_series_sub_number_impl(self, other):
     raise TypingError('{} The object must be a pandas.series or scalar. Given other: {}'.format(_func_name, other))
 
 
+@overload_method(SeriesType, 'sum')
+def hpat_pandas_series_sum(self, axis=None, skipna=None, level=None, numeric_only=None, min_count=0, **kwargs):
```

Comment:
no need to put `kwargs` here
Also, it looks like inside Pandas sources, `skipna` parameter has default `true` (skipna=True). It might be a mistake in Pandas documentation.

Reply:
Inside Pandas:
```
if (skipna == None):
    skipna = True
````